bot/paginatedmessages/paginatedinteractions.go

```diff
@@ -93,11 +95,17 @@ func createNavigationButtons(prevDisabled bool, nextDisabled bool) []discordgo.M
 	}
 }
 
-func CreatePaginatedMessage(guildID, channelID int64, initPage, maxPages int, pagerFunc PagerFunc) (*PaginatedMessage, error) {
+func CreatePaginatedMessage(data *dcmd.Data, guildID, channelID int64, initPage, maxPages int, pagerFunc PagerFunc) (*PaginatedMessage, error) {
```

Comment:
you could make data parameter variadic, then you don't have to edit all function calls all over the place

Reply:
Would making it variadic allow 0 of it to be passed and still work? I've never really used variadic parameters as I typically see no reason for them/

Reply:
https://go.dev/play/p/AU_Xu0_QKLW